app/js/components/tabs/tabsComponent.jsx

```diff
@@ -55,8 +55,9 @@ class TabsComponent extends Component {
             apiHelper.get(url).then(response => {
                 response.json().then(data => {
                     resolve(data);
-                });
-            });
+                //TODO: This errors need to be handled
```

Comment:
Yes, @dkayiwa but not in this ticket. There is currently an open issue that request for the complete refactor of this component [Link to issue](https://issues.openmrs.org/browse/CB-52?jql=project%20%3D%20CB%20and%20status%20!%3D%20closed). Handling this errors should fall under the issue above. Also I would like to recommend we create an issue to handle all components errors in the code base, as currently there is no structure in place to do that.